rclrs/src/node.rs

```diff
@@ -908,6 +910,216 @@ impl NodeState {
         )
     }
 
+    /// Create a [`Timer`] with a repeating callback.
+    ///
+    /// See also:
+    /// * [`Self::create_timer_oneshot`]
+    /// * [`Self::create_timer_inert`]
+    ///
+    /// # Behavior
+    ///
+    /// While the callback of this timer is running, no other callbacks associated
```

Comment:
@mxgrey can you add a comment mentioning that this would be the equivalent of of the `create_timer` API in `rclcpp`? Just as a reference for people coming to `rclrs` from `rclcpp`. Thanks.

Reply:
Sorry for how long it took me to respond, I spent most of September travelling, so this fell deep into my backlog.

[Added here](https://github.com/ros2-rust/ros2_rust/pull/480/commits/52238cf3ee1f9f5f13788b8616f5c48e344f4028#diff-edcb3d03aa3bd1a7a070881f3faa29fc9b06812e2fc26eaeb32e2d431067f268R915).